src/HttpOverStream.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.28606.126
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.329

```

Comment:
Sorry about the churn!

Reply:
No problem, this will just prompt me for updating projects next time I open the solution :D